db/schema.rb

```diff
@@ -570,7 +570,7 @@
     t.integer "visibility", default: 0
     t.bigint "created_by_id", null: false
     t.bigint "updated_by_id", null: false
-    t.jsonb "actions", default: "{}", null: false
+    t.jsonb "actions", default: {}, null: false
```

Comment:
Intentional, found one issue around it while testing.

Reply:
@tejaswinichile  if this was committed in a previous migration. We might need to have an accompanying migration to change this? 

Or is it that the wrong schema file was created?